File: jenkinsScripts/result.py
from os import getenv
s_one = getenv('STAGE_ONE')
s_two = getenv('STAGE_TWO')

if s_one == True and s_two == True:
    print("El proyecto va viento en popa!!!")
elif s_one == False and s_two == False:
    print( "Esto pinta muy mal")
else:
    print("Alguna de las dos stages ha fallado")

File: jenkinsScripts/result.js
const paramA = process.env.STAGE_ONE
const paramB = process.env.STAGE_TWO

if (paramA == false || paramB == false) {
    console.log("Alguna de las dos stages ha fallado")
} else if (paramA == false && paramB == false) {
    console.log( "Esto pinta muy mal")
} else {
    console.log( "El proyecto va viento en popa!!!" )
}
